src/hyperion/experiment_plans/flyscan_xray_centre_plan.py

```diff
@@ -196,12 +197,16 @@ def do_fgs():
         yield from bps.wait()
         LOGGER.info("kicking off FGS")
         yield from bps.kickoff(gridscan, wait=True)
+        gridscan_start_time = time()
         LOGGER.info("Waiting for Zocalo device queue to have been cleared...")
         yield from bps.wait(
             ZOCALO_STAGE_GROUP
         )  # Make sure ZocaloResults queue is clear and ready to accept our new data
         LOGGER.info("completing FGS")
         yield from bps.complete(gridscan, wait=True)
+        LOGGER.info(
+            f"Gridscan motion program took {round(time()-gridscan_start_time,2)} to complete"
+        )
```

Comment:
Should: We will hopefully have proper metrics soon so I don't think we should include printing timings like this

Reply:
It's quite useful to see how long things are taking while we are still increasing the run-up distance. I could just add it in temporarily when I do testing instead?

Reply:
OK, can you leave it in with a comment saying to remove it when we have metrics?